src/js/common/schemaform/FormIntroButtons.jsx

```diff
@@ -25,7 +25,9 @@ class FormIntroButtons extends React.Component {
   handleLoadForm = () => {
     // If successful, this will set form.loadedData.metadata.returnUrl and will
     //  trickle down to this.props to be caught in componentWillReceiveProps
-    this.props.fetchInProgressForm(this.props.formId, this.props.migrations);
+    this.props.fetchInProgressForm(this.props.formId, this.props.migrations)
+      // Should probably do GA or sentry or something, but for now...
+      .catch((e) => console.error('Error loading form:', e)); // eslint-disable-line no-console
```

Comment:
👍 I'll leave it in here for now because it'll be taken out with [your PR](https://github.com/department-of-veterans-affairs/vets-website/pull/5821).